stacks/level2_stack.py

```diff
@@ -0,0 +1,109 @@
+from typing import cast, Any
+
+from aws_cdk import Duration, Stack, RemovalPolicy
+from aws_cdk.aws_iam import Effect, PolicyStatement
+from aws_cdk.aws_lambda import (
+    Architecture,
+    DockerImageCode,
+    DockerImageFunction,
+)
+from aws_cdk.aws_lambda_event_sources import SqsEventSource
+from aws_cdk.aws_sqs import DeadLetterQueue, Queue
+from constructs import Construct
+
+DOCKER_IMAGE = """
+FROM odinsmr/u-jobs:qsmr_{invmode}_{freqmode}_{project}
+
+FROM public.ecr.aws/lambda/python:3.9
```

Comment:
This looks a bit strange, this looks like a multistage build but no artefact is brought from the first image to the second. So I guess you will not have anything to work with

usually something is taken from the first image like this:
```
FROM odinsmr/u-jobs:qsmr_{invmode}_{freqmode}_{project} as builder
RUN something
FROM public.ecr.aws/lambda/python:3.9
COPY --from=builder /datafile /var/data/datafile
...
```
or in this case when we just want to pick some files (many?)
```
FROM public.ecr.aws/lambda/python:3.9
COPY --from=odinsmr/u-jobs:qsmr_{invmode}_{freqmode}_{project} /datafile /var/data/datafile
...
```

Reply:
Typ så? https://github.com/Odin-SMR/odin-l2-lambda/pull/1/commits/04d9acaaa71595ea424060991b826c32b29eafa0